WordPress/src/main/java/org/wordpress/android/ui/engagement/LikedItemViewHolder.kt

```diff
@@ -19,25 +23,23 @@ class LikedItemViewHolder(
     parent: ViewGroup,
     private val imageManager: ImageManager
 ) : EngagedPeopleViewHolder(parent, R.layout.note_block_header) {
-    private val name = itemView.findViewById<TextView>(R.id.header_user)
     private val snippet = itemView.findViewById<TextView>(R.id.header_snippet)
     private val avatar = itemView.findViewById<ImageView>(R.id.header_avatar)
-    private val rootView = itemView.findViewById<View>(R.id.header_root_view)
+    private val rootView = itemView.findViewById<CardView>(R.id.header_root_view)
 
-    fun bind(likedItem: LikedItem) {
+    fun bind(likedItem: LikedItem, type: ListScenarioType) {
         val authorName = when (val author = likedItem.author) {
             is AuthorNameString -> author.nameString
             is AuthorNameCharSequence -> author.nameCharSequence
         }
 
-        this.name.text = authorName
         this.snippet.text = likedItem.postOrCommentText
 
         val avatarUrl = WPAvatarUtils.rewriteAvatarUrl(
             likedItem.authorAvatarUrl,
-            rootView.context.resources.getDimensionPixelSize(R.dimen.avatar_sz_small)
+            rootView.context.resources.getDimensionPixelSize(R.dimen.avatar_sz_extra_small)
         )
-
+        avatar.isGone = type == ListScenarioType.LOAD_POST_LIKES
```

Comment:
I noticed that the avatar is also shown in the comment details. I'm not suggesting a change now just noting it down to change this when we work on that screen.

![Screenshot_20240424_092023](https://github.com/wordpress-mobile/WordPress-Android/assets/304044/f8c41a7a-43bf-492f-9bcf-d3070b68819c)

Reply:
Nice catch! I've updated the visibility of the avatar, but I'll take a deep look at the implementation when I develop the task of 🤖 Comment Details: Header redesign